cmd/kubeadm/app/cmd/phases/preflight.go

```diff
@@ -22,14 +22,35 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
 	"k8s.io/kubernetes/cmd/kubeadm/app/preflight"
+	"k8s.io/kubernetes/pkg/util/normalizer"
+)
+
+var (
+	masterPreflightLongDesc = normalizer.LongDesc(`
+		Run master pre-flight checks, functionally equivalent to what implemented by kubeadm init.
+		` + cmdutil.AlphaDisclaimer)
+
+	masterPreflightExample = normalizer.Examples(`
+		# Run master pre-flight checks.
+		kubeadm alpha phase preflight master
+		`)
+
+	nodePreflightLongDesc = normalizer.LongDesc(`
+		Run node pre-flight checks, functionally equivalent to what implemented by kubeadm join.
+		` + cmdutil.AlphaDisclaimer)
+
+	nodePreflightExample = normalizer.Examples(`
+		# Run node pre-flight checks.
+		kubeadm alpha phase preflight node
+	`)
 )
 
 // NewCmdPreFlight calls cobra.Command for preflight checks
 func NewCmdPreFlight() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "preflight",
 		Short: "Run pre-flight checks",
-		RunE:  cmdutil.SubCmdRunE("preflight"),
```

Comment:
Why remove this? I think we need it so that `kubeadm alpha phase preflight` alone exits 1

Reply:
Because if you define `RunE` then `kubeadm alpha phase preflight` output is:
```
Error: missing subcommand; "preflight" is not meant to be run on its own
Usage:
  kubeadm alpha phase preflight [flags]
  kubeadm alpha phase preflight [command]
```

Without `RunE`  instead `kubeadm alpha phase preflight`  outputs: 
```
Usage:
  kubeadm alpha phase preflight [command]
```

and this is cleaner and without the wrong `kubeadm alpha phase preflight [flags]` usage

Reply:
ok, fine with this then as is,  but @fabriziopandini please create an issue for reviewing all these Run/RunE's so we can enforce consistency across the board